dashboard/app/views/layouts/_share_footer.html.haml

```diff
@@ -0,0 +1,41 @@
+#page-small-footer
+
+:ruby
+  reactProps = {
+    i18nDropdown: '',
+    copyrightInBase: false,
+    baseMoreMenuString: I18n.t('footer.built_on_code_studio'),
+    className: 'dark',
+    menuItems: [
+      {
+        text: I18n.t('footer.make_my_own_app'),
+        link: 'https://code.org/learn',
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.how_it_works'),
+        link: url_for(action: 'edit'),
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.copyright'),
+        link: '#',
+        copyright: true
+      },
+      {
+        text: I18n.t('footer.tos'),
+        link: "https://code.org/tos",
+        newWindow: true
+      },
+      {
+        text: I18n.t('footer.privacy'),
+        link: "https://code.org/privacy",
+        newWindow: true
+      }
+    ],
+    copyrightStrings: build_copyright_strings
+  }
```

Comment:
Talked this over offline.  One option would be to pass in `footer_menu_options` instead of a `share_footer` boolean, but this is mostly a style choice and the code is fine to merge as-is.